package.json

```diff
@@ -25,14 +25,12 @@
     "url": "https://github.com/tesselo/vue2-leaflet-vectorgrid/issues"
   },
   "homepage": "https://github.com/tesselo/vue2-leaflet-vectorgrid#readme",
-  "dependencies": {
-    "leaflet.vectorgrid": "^1.3.0"
-  },
   "devDependencies": {
     "poi": "^9.3.10"
   },
   "peerDependencies": {
-    "leaflet": "^1.3.1",
-    "vue2-leaflet": "^1.0.1"
+    "leaflet": "^1.3.4",
+    "vue2-leaflet": "^2.0.2",
```

Comment:
Were there any big breaking changes in vue2-leaflet 2.0?

Reply:
Yes, components import split and leaflet moved in peerDependencies. [See here](https://github.com/KoRiGaN/Vue2Leaflet#breaking-change-from-1xx-to-2xx)